frontend/public/components/cluster-settings/global-config.tsx

```diff
@@ -3,21 +3,29 @@ import * as _ from 'lodash-es';
 import { connect } from 'react-redux';
 import { Link } from 'react-router-dom';
 
-import { K8sKind, k8sList } from '../../module/k8s';
+import { K8sKind, k8sList, getStoredSwagger, SwaggerDefinition, SwaggerDefinitions } from '../../module/k8s';
 import { resourcePathFromModel } from '../utils/resource-link';
 import { LoadingBox } from '../utils/status-box';
 
+const allDefinitions: SwaggerDefinitions = getStoredSwagger();
+
 const globalConfigListStateToProps = ({k8s}) => ({
   configResources: k8s.getIn(['RESOURCES', 'configResources']),
 });
 
 const ItemRow = ({item}) => {
   const resourceLink = resourcePathFromModel(item.model, item.metadata.name, item.metadata.namespace);
+  const itemPath = [Object.keys(allDefinitions).find(key => key.endsWith(`openshift.config.${item.model.apiVersion}.${item.kind}`))];
```

Comment:
You should be able to use the helper from #1676

Reply:
Nice!  Updated.